results/energy.tex

```diff
@@ -0,0 +1,142 @@
+\section{Energy Consumption}
+
+In this section we look at the energy consumption of the Sensor Tracker application, described in \autoref{sec:impl:project:ii}.
+The Sensor Tracker is an interrupt driven application and has two modes of operation from an energy consumption perspective.
+It turns \emph{On} for each interrupt to perform the measurement and turns \emph{Off} when the measuring if done.
+
+\subsection{Measuring}
+The measurement was performed with the \emph{Energy Profiler} application supplied by Silicon Labs in their \emph{Simplicity Studio} software suite.
+The profiler measures on board current at a sample frequency of 6250Hz.
+Power, measured in J/s, is produced by the formula $P (J/s) = V*I $.
+By accumulating the Power over time we get the energy consumption given in \emph{Joule}.
+This metric is reported by the \emph{Energy Profiler}.
+
+Each measurement, called a \emph{run}, was gathered manually by executing the sample collection process (see \autoref{sec:project:i:sample-collection}) for 30s and recording the energy reported by the profiler.
+This was repeated 4 times for each data point and an average was calculated after removing the sample with the highest variance.
+This process was introduced to remove human error from the manual collection.
+The number of collected and discarded samples was based upon the stability of the results, and the largest variance in the collected samples are 0.174\%.
+
+\subsection{Parameter}
+
+The energy consumption was recorded for the Sensor Tracker, which was configured with different workloads.
+The workload is dependent on how many interrupts are trigger per run and how much work is performed to at each interrupt.
```

Comment:
"... per run`, ` and on how much work is performed ~~to at~~ `during each` interrupt"